src/pages/IncidentTimeline/IncidentTimeline.scss

```diff
@@ -0,0 +1,29 @@
+@import "../../theme";
+
+.timeline-container {
+  display: flex;
+  flex-direction: row;
+  min-height: 93vh;
+
+  .timeline-sidebar {
+    justify-content: flex-start;
+    z-index: 2;
+    position: relative;
+  }
+
+  .timeline-main-content {
+    justify-content: flex-end;
+    width: 100vw;
+    padding: 0 1rem;
+    z-index: 0;
+    position: relative;
+
+      .chat-tab {
```

Comment:
Why do we have `classes defined with no styling?` If the styling is to be added later, kindly provide a comment or any form of directive on these classes.

Reply:
On it

Reply:
fixed